x/wasm/internal/types/msg.go

```diff
@@ -1,17 +1,27 @@
 package types
 
 import (
+	"fmt"
+	"regexp"
+
+	"github.com/asaskevich/govalidator"
+
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
 const (
-	MaxWasmSize = 500 * 1024
+	MaxWasmSize   = 500 * 1024
+	BuildTagRegex = "cosmwasm-opt:"
```

Comment:
Maybe `"^cosmwasm-opt:"` so it can't start with other characters? Otherwise, that seems fine enough... some limits but not terribly restrictive